tests/integration/polygonParts/polygonParts.spec.ts

```diff
@@ -47,6 +59,7 @@ describe('polygonParts', () => {
 
   afterAll(async () => {
     await helperDB.destroyConnection();
+    //await deleteDB(testDataSourceOptions);
```

Comment:
Why commented?

Reply:
because when running locally, it is better to add this line to delete the created db after all tests, instead of removing it manually after each run. BUT it is dangerous to leave it here if someone will run this( as plain code or with docker)  on the azure or inside networks, this can delete the DB. 
so i left it here to make the next developer life easier , maybe add a comment that explains this?